lib/runners/processor/metrics_fileinfo.rb

```diff
@@ -85,7 +85,7 @@ def last_committed_at
 
     def analyze_last_committed_at(targets)
       trace_writer.message "Analyzing last commit time..." do
-        targets.each do |target|
+        Parallel.each(targets, in_threads: Parallel.processor_count) do |target|
```

Comment:
[suggestion] `Parallel.processor_count` seems unstable because of `Etc.nprocessors`:

https://github.com/grosser/parallel/blob/875c9f472e6da3450219a7ba2d29bf430f23adfe/lib/parallel/processor_count.rb#L7-L9

How about explicitly specifying the value like this? ⬇️ 

```suggestion
        Parallel.each(targets, in_threads: 2) do |target|
```